osversion/windowsbuilds.go

```diff
@@ -35,4 +35,8 @@ const (
 
 	// V20H2 corresponds to Windows Server 20H2 (semi-annual channel).
 	V20H2 = 19042
+
+	// minimum build required for cimfs
+	//TODO(ambarve): replace this with correct value before merging
+	IRON_BUILD = 21280
```

Comment:
I know this is just a placeholder until we know the build but we might as well stick with the naming scheme, so this should be V21H1